CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.0.0-rc.2 (2020-08-20)
+
+- Updated the toolbar to be pinned at the top of the page when the data manager is visible. [#10](https://github.com/blackbaud/skyux-data-manager/pull/10)
+- Fixed sticky settings to only load the data state once. [#10](https://github.com/blackbaud/skyux-data-manager/pull/10)
+- Updated `<sky-data-manaager-toolbar-left-item>` and `<sky-data-manaager-toolbar-left-right>` elements to use one per item to render in the toolbar. [#10](https://github.com/blackbaud/skyux-data-manager/pull/10)
```

Comment:
Manager is misspelled in both instances.  

I'm a little unclear with current wording because I'm unsure what we're using one of. Does this change the meaning?
Updated `<sky-data-manager-toolbar-left-item>` and `<sky-data-manager-toolbar-left-right>` to use one element per item to render in the toolbar.